dwitter/feed/views.py

```diff
@@ -169,20 +230,23 @@ def get_dweet_list(self):
 
 class NewUserFeed(UserFeed):
     sort = SortMethod.NEW
+    feed_name = "new"
 
     def get_title(self):
         return "Dweets by u/" + self.kwargs['url_username'] + " | Dwitter"
 
 
 class TopUserFeed(UserFeed):
     sort = SortMethod.TOP
+    feed_name = "top"
```

Comment:
How will this view handle not having options for week, month etc.?

Reply:
Just fine; the whole top menu has a separate code path for the user menu, the hashtag menu and the dweet feed menu; since the links need to be different.

I did catch a css bug thanks to your comment though, the "top" feed of a user didn't correctly bold "top". Fixed.